dynwalls/__init__.py

```diff
@@ -16,16 +16,16 @@
     SYSTEMD_DIR = os.environ.get("HOME") + "/.local/share/systemd/user"
 
 try:
-	if not os.path.isdir(DATA_DIR) and not os.path.islink(DATA_DIR):
```

Comment:
For I don't know what reason, there were tabs over spaces. So I only changed that.

Reply:
Good, I think sticking to 4 spaces for indentation is best.